<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>RAT</author>
 <class>Euler_Hamilton_Widget</class>
 <widget class="QWidget" name="Euler_Hamilton_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="IODataBox">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="title">
      <string>Nạp đồ thị</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="loadFileButton">
        <property name="text">
         <string>Nạp file chứa đồ thị</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="adjacenyMatrixRB">
        <property name="text">
         <string>Ma trận kề</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dataStructures</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="graphTypesLabel">
        <property name="text">
         <string>Dạng đồ thị</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dataStructuresLabel">
        <property name="text">
         <string>Cấu trúc mô tả đồ thị</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="listRB">
        <property name="text">
         <string>Danh sách cạnh cung</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">dataStructures</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>182</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>182</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="filePathLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="undirectedRB">
        <property name="text">
         <string>Vô hướng</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">graphTypes</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="directedRB">
        <property name="text">
         <string>Có hướng</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">graphTypes</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ProcessDataBox">
     <property name="title">
      <string>Thao tác với thuật toán và đồ thị</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="eulerTab">
         <attribute name="title">
          <string>Euler</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="infoEulerButton">
            <property name="text">
             <string>Thông tin cơ bản</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="analysisEulerButton">
            <property name="text">
             <string>Phân tích - Đánh giá</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="appEulerButton">
            <property name="text">
             <string>Ứng dụng </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="solveEulerButton">
            <property name="text">
             <string>Chu trình/đường đi Euler</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>323</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" rowspan="5">
           <widget class="QTextBrowser" name="textBrowserEuler"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="hamiltonTab">
         <attribute name="title">
          <string>Hamilton</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="infoHalButton">
            <property name="text">
             <string>Thông tin cơ bản</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="analysisHalButton">
            <property name="text">
             <string>Phân tích - Đánh giá</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="appHalButton">
            <property name="text">
             <string>Ứng dụng </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="solveHalButton">
            <property name="text">
             <string>Chu trình/đường đi Hamilton</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="solveTravellingButton">
            <property name="text">
             <string>Bài toán người du lịch</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>294</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" rowspan="6">
           <widget class="QTextBrowser" name="textBrowserHal"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="dataStructures"/>
  <buttongroup name="graphTypes"/>
 </buttongroups>
</ui>
